.NET application update (Overview, Learn about .NET & Get Started) (#1698)
Samsung/tizen-docs: docs/application/dotnet/get-started/first-app.md

To run the application, you must first deploy it to the target: either a device or an emulator. Deploying means transferring the package file (`.tpk`) to the target and invoking the Tizen Package Manager to install it.

> To deploy and run the application on the emulator, you can use the following tools:

Reviewer: ```suggestion
To deploy and run the application on the emulator, follow the steps below:
```
